src/__tests__/expectsToThrow.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-console */
+const expectToThrow = (func) => {
```

Comment:
Just wondering what's the point of that one? Is it just to catch the console error and avoid displaying it?
If so, I'd make the name of that a bit more descriptive

Reply:
Yes, to avoid seeing a big red output when it's actually expected... I thought the name was good, bu added the "NoConsole" suffix and moved it to `testUtils`.